### PR TITLE
Added utilities functions to get explore url for chain and get chain from asset type.

### PR DESCRIPTION
**Pull request by SemyonL95**

### CODE REVIEW

File: coin/models_test.go [hunk 1]
```diff
@@ -37,3 +37,48 @@ func TestGetCoinForId(t *testing.T) {
...
+		{
+			name: "Test ethereum",
+			args: args{
+				addr:  "token",
+				chain: "ethereum",
+			},
+			want:    "https://etherscan.io/token/token",
+			wantErr: false,
+		},
+		{
```
Comment (vikmeup) [inline coin/models_test.go:62]: can you add couple of more test cases here? for tron, since it's using two explorers

File: coin/models.go [hunk 2]
```diff
@@ -15,65 +15,65 @@ func GetCoinForId(id string) (Coin, error) {
 	return Coin{}, errors.New("unknown id: " + id)
 }
 
-func GetCoinExploreURL(chain, addr string) (string, error) {
-	switch strings.ToLower(chain) {
-	case Coins[ETHEREUM].Handle:
+func GetCoinExploreURL(c Coin, addr string) (string, error) {
+	switch c {
```
Comment (vikmeup) [inline coin/models.go:19]:
```suggestion
	switch c.id {
	case coin.ETHEREUM:
	    break
	}
```

Comment (vikmeup) [reply coin/models.go:19]: this would be safer this way

File: coin/models.go [hunk 3]
```diff
@@ -10,3 +14,66 @@ func GetCoinForId(id string) (Coin, error) {
 	}
 	return Coin{}, errors.New("unknown id: " + id)
 }
+
+func GetCoinExploreURL(c Coin, addr string) (string, error) {
```
Comment (vikmeup) [inline coin/models.go:18]:
```suggestion
func GetCoinExploreURL(c Coin, address string) (string, error) {
```

File: types/token_type.go [hunk 4]
```diff
@@ -30,10 +30,19 @@ const (
...
 	AVALANCHE TokenType = "AVALANCHE"
-	FANTOM 	  TokenType = "FANTOM"
+	FANTOM    TokenType = "FANTOM"
 	HRC20     TokenType = "HRC20"
 	ARBITRUM  TokenType = "ARBITRUM"
 	TERRA     TokenType = "TERRA"
+	EOS       TokenType = "EOS"
+	NEP5      TokenType = "NEP5"
+	NRC20     TokenType = "NRC20"
+	VET       TokenType = "VET"
```
Comment (vikmeup) [inline types/token_type.go:40]:
```suggestion
	VIP180       TokenType = "VIP180"
```

File: types/token_type.go [hunk 5]
```diff
@@ -30,10 +30,19 @@ const (
...
+	ONTOLOGY  TokenType = "ONTOLOGY"
+	THETA     TokenType = "THETA"
+	TOMO      TokenType = "TOMO"
```
Comment (vikmeup) [inline types/token_type.go:43]:
```suggestion
	TRC20      TokenType = "TRC20"
```

File: types/token_type.go [hunk 6]
```diff
@@ -30,10 +30,19 @@ const (
...
+	WAVES     TokenType = "WAVES"
+	POA       TokenType = "POA"
```
Comment (vikmeup) [inline types/token_type.go:45]:
```suggestion
	POA20       TokenType = "POA20"
```

File: types/chain.go [hunk 7]
```diff
@@ -0,0 +1,65 @@
+package types
+
+import (
+	"errors"
+	"github.com/trustwallet/go-primitives/coin"
+)
+
+func GetChainFromAssetType(type_ string) (coin.Coin, error) {
```
Comment (leedaniil) [inline types/chain.go:8]: Avoid underscores `_` except for unit test. You call name it like `assetType` for example.

File: types/chain.go [hunk 8]
```diff
@@ -0,0 +1,65 @@
+package types
+
+import (
+	"errors"
+	"github.com/trustwallet/go-primitives/coin"
```
Comment (leedaniil) [inline types/chain.go:5]: Can you split packages? Separate group for built-in package, separate group for project packages and separate group for third-party packages.

For example:
```
import (
        "errors"
        "strings"

        "github.com/trustwallet/go-primitives/coin"

        "some/external/package/here"
        ""
)
```

Comment (SemyonL95) [reply types/chain.go:5]: Sorry, don't understand what is built-in packages in your case? should I move it to an already existed package or create a new "chain"?

Comment (vikmeup) [reply types/chain.go:6]: Just use `goimports` in your IDE

**Pull request merged**